### DISCUSSION TITLE
Tensorflow with GPU server

### DISCUSSION THREAD

**Discussion by shivareddyiirs** [microsoft/PlanetaryComputer#58] [Q&A]
I am not able to start server.

![image](https://user-images.githubusercontent.com/5653512/167546302-6d718275-b5de-4f5b-8796-ad1d77e8fdbc.png)

**Comment by TomAugspurger** [comment]
We don't have a great error message right now, but you're likely trying to spawn a GPU node when all of our nodes are already in use.

You can either try again later, try a non-GPU node, or [deploy your own compute](https://planetarycomputer.microsoft.com/docs/concepts/computing/)
